lib/npm-check-updates.js

```diff
@@ -103,14 +103,17 @@ function createDependencyTable() {
  * @param {object} args
  * @param {object} args.from
  * @param {object} args.to
+ * @param {object} args.owner
+ * @param {object} options
  * @returns {Table}
  */
-function toDependencyTable(args) {
+function toDependencyTable(args, options) {
```

Comment:
I believe `args.owner[dep]` will be `undefined` if `options.ownerChanged` is `false`, so there is no need to check it again here. Thus you can avoid having to pass the options object to `toDependencyTable`.